packages/core/src/components/Tooltip/index.tsx

```diff
@@ -34,6 +36,8 @@ export type TooltipProps = {
   onClose?: () => void;
   /** Callback fired when the tooltip is shown. */
   onShow?: () => void;
+  /** True to enable interactive popover functionality */
```

Comment:
super nit but all of these should be full sentences / end in `.` 😬

Reply:
missing `.`? what missing `.`? (i'll add the period in a later tooltip addition, don't want to wait for CI again 😭 

![image](https://user-images.githubusercontent.com/17525561/86182824-85020280-bae5-11ea-9687-fdb434216219.png)